packages/react-router-dom/docs/guides/server-rendering.md

```diff
@@ -2,7 +2,7 @@
 
 Rendering on the server is a bit different since it's all stateless. The basic idea is that we wrap the app in a stateless [`<StaticRouter>`][staticrouter] instead of a [`<BrowserRouter>`][browserrouter]. We pass in the requested url from the server so the routes can match and a `context` prop we'll discuss next.
 
-```jsx
+```js
```

Comment:
I'm not sure the heuristic you're using for switching parsers (many blocks above are still `jsx`), but I think for JSX-heavy code like this, the `jsx` tokenization comes out better than `js`, mostly for props visibility/contrast:

![image](https://user-images.githubusercontent.com/17882/66364069-6e63bc00-e956-11e9-8b53-4cd5f6a8df4f.png)

![image](https://user-images.githubusercontent.com/17882/66364087-80ddf580-e956-11e9-9ad6-f6c6d9bc3a10.png)